python-avd/pyavd/_eos_designs/structured_config/network_services/port_channel_interfaces.py

```diff
@@ -121,6 +121,7 @@ def _set_l3_port_channels(
                     mtu=self.shared_utils.get_interface_mtu(l3_port_channel.name, l3_port_channel.mtu),
                     description=interface_description or None,
                     ip_address=l3_port_channel.ip_address,
+                    ip_address_secondaries=l3_port_channel.ip_address_secondaries._cast_as(EosCliConfigGen.PortChannelInterfacesItem.IpAddressSecondaries),
```

Comment:
Following my discussion with @ClausHolbechArista, this should work and be slighty more efficient than `_cast_as` (less checks):

```suggestion
                    ip_address_secondaries=EosCliConfigGen.PortChannelInterfacesItem.IpAddressSecondaries(l3_port_channel.ip_address_secondaries)
```